index.js

```diff
@@ -1,3 +1,8 @@
+// Guard against document not being defined in non-browser environments.
+if (typeof document !== 'undefined') {
```

Comment:
should this not be ` typeof global.document` ? Since we always access via global variable